Index: getrandom/Cargo.toml
===================================================================
--- getrandom.orig/Cargo.toml
+++ getrandom/Cargo.toml
@@ -42,8 +42,6 @@ dummy = []
 rustc-dep-of-std = ["compiler_builtins", "core"]
 std = []
 test-in-browser = ["wasm-bindgen"]
-[target."cfg(target_os = \"wasi\")".dependencies.wasi]
-version = "0.9"
 [target."cfg(unix)".dependencies.libc]
 version = "0.2.64"
 default-features = false
