Index: git2/Cargo.toml
===================================================================
--- git2.orig/Cargo.toml
+++ git2/Cargo.toml
@@ -60,7 +60,6 @@ https = ["libgit2-sys/https", "openssl-s
 ssh = ["libgit2-sys/ssh"]
 ssh_key_from_memory = ["libgit2-sys/ssh_key_from_memory"]
 unstable = []
-vendored-openssl = ["openssl-sys/vendored"]
 [target."cfg(all(unix, not(target_os = \"macos\")))".dependencies.openssl-probe]
 version = "0.1"
 optional = true
