Index: globset/Cargo.toml
===================================================================
--- globset.orig/Cargo.toml
+++ globset/Cargo.toml
@@ -45,9 +45,9 @@ version = "1.1.5"
 features = ["perf", "std"]
 default-features = false
 
-[dependencies.serde]
-version = "1.0.104"
-optional = true
+#[dependencies.serde]
+#version = "1.0.104"
+#optional = true
 [dev-dependencies.glob]
 version = "0.3.0"
 
@@ -58,5 +58,5 @@ version = "1"
 version = "1.0.45"
 
 [features]
-serde1 = ["serde"]
+#serde1 = ["serde"]
 simd-accel = []
