--- a/build.rs
+++ b/build.rs
@@ -17,7 +17,7 @@
     // The system copy of libssh2 is not used by default because it
     // can lead to having two copies of libssl loaded at once.
     // See https://github.com/alexcrichton/ssh2-rs/pull/88
-    if env::var("LIBSSH2_SYS_USE_PKG_CONFIG").is_ok() {
+    if true {
         if let Ok(lib) = pkg_config::find_library("libssh2") {
             for path in &lib.include_paths {
                 println!("cargo:include={}", path.display());
@@ -26,7 +26,7 @@
         }
     }
 
-    if !Path::new("libssh2/.git").exists() {
+    if false {
         let _ = Command::new("git")
             .args(&["submodule", "update", "--init"])
             .status();
