Index: lazycell/Cargo.toml
===================================================================
--- lazycell.orig/Cargo.toml
+++ lazycell/Cargo.toml
@@ -21,9 +21,6 @@ readme = "README.md"
 keywords = ["lazycell", "lazy", "cell", "library"]
 license = "MIT/Apache-2.0"
 repository = "https://github.com/indiv0/lazycell"
-[dependencies.clippy]
-version = "0.0"
-optional = true
 
 [dependencies.serde]
 version = "^1"
@@ -31,4 +28,3 @@ optional = true
 
 [features]
 nightly = []
-nightly-testing = ["clippy", "nightly"]
