diff --git a/vendor/unicode-xid/src/lib.rs b/vendor/unicode-xid/src/lib.rs
index 7dd95ba65..fcda2be2f 100644
--- a/vendor/unicode-xid/src/lib.rs
+++ b/vendor/unicode-xid/src/lib.rs
@@ -38,8 +38,6 @@
 //! ```
 
 #![deny(missing_docs, unsafe_code)]
-#![doc(html_logo_url = "https://unicode-rs.github.io/unicode-rs_sm.png",
-       html_favicon_url = "https://unicode-rs.github.io/unicode-rs_sm.png")]
 
 #![no_std]
 #![cfg_attr(feature = "bench", feature(test, unicode_internals))]
