--- /dev/null
+++ b/build.rs
@@ -0,0 +1,9 @@
+use std::env;
+
+pub fn main() {
+    if let Ok(os) = env::var("CARGO_CFG_TARGET_OS") {
+        if os == "redox" {
+            println!("cargo:rustc-cfg=nightly");
+        }
+    }
+}
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -1,5 +1,5 @@
 #![feature(asm)]
-#![feature(llvm_asm)]
+#![cfg_attr(nightly, feature(llvm_asm))]
 #![cfg_attr(not(test), no_std)]
 
 #[cfg(test)]
@@ -10,6 +10,7 @@
 pub use self::data::*;
 pub use self::error::*;
 pub use self::flag::*;
+#[cfg(target_os = "redox")]
 pub use self::io::*;
 pub use self::number::*;
 pub use self::scheme::*;
@@ -51,6 +52,7 @@
 pub mod flag;
 
 /// Functions for low level hardware control
+#[cfg(target_os = "redox")]
 pub mod io;
 
 /// Call numbers used by each system call
