--- a/Cargo.toml
+++ b/Cargo.toml
@@ -63,13 +63,10 @@
 default = ["ssl"]
 force-system-lib-on-osx = ["curl-sys/force-system-lib-on-osx"]
 http2 = ["curl-sys/http2"]
-protocol-ftp = ["curl-sys/protocol-ftp"]
-spnego = ["curl-sys/spnego"]
 ssl = ["openssl-sys", "openssl-probe", "curl-sys/ssl"]
 static-curl = ["curl-sys/static-curl"]
 static-ssl = ["curl-sys/static-ssl"]
 upkeep_7_62_0 = ["curl-sys/upkeep_7_62_0"]
-zlib-ng-compat = ["curl-sys/zlib-ng-compat", "static-curl"]
 [target."cfg(all(unix, not(target_os = \"macos\")))".dependencies.openssl-probe]
 version = "0.1.2"
 optional = true
